README.md

```diff
@@ -57,35 +71,68 @@ it('should return `help message` if user types `help`', () => {
                 ]
             }
         ]
-    ).then((message) => {
-        // In message, we receive a full object that includes params:
-        // {
-        //    user: 'someUserId',
-        //    channel: 'someChannel',
-        //    text: 'help message',
-        // }
-        return assert.equal(message.text, 'help message');
-    })
+    );
+    /* example of botkit response
+    { type: 'message',
+      text: 'help message',
+      attachmentLayout: undefined,
+      attachments: undefined,
+      suggestedActions: undefined,
+      speak: undefined,
+      inputHint: undefined,
+      summary: undefined,
+      textFormat: undefined,
+      importance: undefined,
+      deliveryMode: undefined,
+      expiration: undefined,
+      value: undefined,
+      channelData:
+       { channelId: 'slack',
+         serviceUrl: '',
+         conversation: { id: 'someChannel', thread_ts: null },
+         from: { id: '' },
+         recipient: { id: 'someUserId' } },
+      channelId: 'slack',
+      serviceUrl: undefined,
+      conversation: { id: 'someChannel', thread_ts: null },
+      from: { id: undefined },
+      recipient: { id: 'someUserId' } 
+      }
+     */
+    return assert.equal(message.text, 'help message');
 });
 ```
-## Advanced Usage ##
 
-`botExtender` - allows developers to extend the bot and add custom functions to the `bot`. This overrides functionality in [BotmockWorker.js](https://github.com/gratifychat/botkit-mock/blob/migrate_to_botkit_core/lib/BotmockWorker.js).
+## Advance usage
+`Botkit` itself depends on adapters (slack, facebook, etc.).
+`Botmock` is an extension of `Botkit` provides interface for accepting user messages through `.usersInput`.
+You could connect any botkit providers to botmock.
+For now, we have only advance slack binding for the [slack provider](https://github.com/howdyai/botkit/tree/master/packages/botbuilder-adapter-slack). 
```

Comment:
What file is this mocked in?

Reply:
There is no actual file with full slack mock as in a previous `botkit-mock`, with a new botkit structure we use a package `botbuilder-adapter-slack` it is standalone solution.
Generally we have `SlackApiMock` that helps us to bind only mocked api calls to `slack api` over `botbuilder-adapter-slack` so we do not implement by ourself bot methods.